lib/SimpleSAML/Metadata/SAMLParser.php

```diff
@@ -124,6 +124,11 @@ class SimpleSAML_Metadata_SAMLParser
      */
     private $entityAttributes;
 
+    /**
+     * An associative array of attributes from the RegistrationInfo element.
+     * @var array
+     */
+    private $registrationInfo;
```

Comment:
Ported this to
https://github.com/simplesamlphp/simplesamlphp/pull/335

If it conflicts with this PR, then I can remove the change from this PR once 335 is approved.